### PR TITLE
Update FilterHelper.py

### PR DESCRIPTION
**Pull request by ayush1999**
Fixes Issue #4497

### CODE REVIEW

File: coalib/parsing/FilterHelper.py [hunk 1]
```diff
@@ -12,24 +12,23 @@ class FilterHelper:
...
-                get_all_bears)
-            all_bears = get_all_bears(LogPrinter())
-        if not cls.is_valid_filter(filter):
-            raise InvalidFilterException('{!r} is an invalid filter. '
-                                         'Available filters: {}'.format(
-                                             filter, cls.get_all_filters_str()))
-        if not args or len(args) == 0:
-            return all_bears
-        return cls.available_filters[filter](all_bears, args)
+def get_all_filters_str(cls, sep=', '):
```
Comment (shreyans800755) [inline coalib/parsing/FilterHelper.py]: We don't need `cls` argument as this is no more `classmethod`.

File: coalib/parsing/FilterHelper.py [hunk 2]
```diff
@@ -12,24 +12,23 @@ class FilterHelper:
...
+    return sep.join(sorted(cls.available_filters))
+
+
+def is_valid_filter(cls, filter):
```
Comment (shreyans800755) [inline coalib/parsing/FilterHelper.py]: Remove `cls`

File: coalib/parsing/FilterHelper.py [hunk 3]
```diff
@@ -12,24 +12,23 @@ class FilterHelper:
...
+    return filter in cls.available_filters
+
+
+def get_filtered_bears(cls, filter, args, all_bears=None):
```
Comment (shreyans800755) [inline coalib/parsing/FilterHelper.py]: Remove `cls`

**Comment by shreyans800755** [comment]
@ayush1999 Amend the original commit instead of pushing more commits. These PR only requires a single commit at the moment.

**Comment by shreyans800755** [comment]
@ayush1999 You need to update the references of these file in import to get all the functions directly from module scope instead of class. That's why tests are failing.
Try to pass all the tests and also running `coala` in local before pushing to reduce review/change iterations.

**Comment by ayush1999** [comment]
I'll make all changes in my fork of the PR and update the PR in couple of days.

**Pull request closed without merging**